components/base/Button.tsx

```diff
@@ -127,14 +126,13 @@ const StyledLink = styled.a`
 `;
 
 const ButtonLink = ({ text, size, variant, link }: Props) => (
-  <Link href={link.href} passHref>
-    <StyledLink
-      target={link.external && "_blank"}
-      className={`${variant} ${size}`}
-    >
-      {text}
-    </StyledLink>
-  </Link>
+  <StyledLink
+    href={link.href}
+    target={link.external && "_blank"}
+    className={`${variant} ${size}`}
+  >
+    {text}
+  </StyledLink>
```

Comment:
Removed <Link> from buttons since they're all external links. If in the future we need to add Buttons that link internally, we can add a conditional wrapper.